scalajvm/app/controllers/Quotes.scala

```diff
@@ -21,6 +23,43 @@ object Quotes extends Controller {
     Ok(views.html.index(quotes, pageNumber, countPages, order, filter))
   }
 
+  def getRSSFeed = ActionWithTx { implicit request =>
+    import request.dbSession
+    import play.api.Play.current
+
+    val feedLimit = current.configuration.getInt("feed.limit").get
+
+    val quotes = QuoteQueries().getPageOfQuotes(0, feedLimit,
+        QuoteOrdering.Time, QuoteFilter.None)
+    val items = quotes.map(quote => {
+        val description = quote.content getOrElse ""
+        val cdataReady = description.replaceAll("]]>", "]]]]><![CDATA[>")
```

Comment:
On line 45, I'm packing the description into CDATA. This line "escapes" characters that might break escaped version of description. See [Stack Overflow answer](http://stackoverflow.com/a/223782).